### PR TITLE
LPS-186955 [FE] Unique Composed Key

### PR DESCRIPTION
**Pull request by igor-franca**
[Related](https://liferay.atlassian.net/browse/LPS-186955)

### CODE REVIEW

**Comment by igor-franca** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalDeletionNotAllowed.tsx [hunk 1]
```diff
@@ -5,53 +5,67 @@
...
 }
 
-export function ModalDeletionNotAllowed({
+function ModalDeletionNotAllowed({
+	content,
 	onVisibilityChange,
-	selectedItemLabel,
 }: ModalDeletionNotAllowedProps) {
+	const [visibility, setVisibility] = useState(false);
+	const [bodyContent, setBodyContent] = useState<React.ReactNode>(content);
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalDeletionNotAllowed.tsx:20]: sort

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/UniqueComposedKey.tsx [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+				objectField.externalReferenceCode ===
+					objectValidationRuleSetting.value &&
+				objectValidationRuleSetting.name ===
+					objectValidationRuleSettingNameMatches
+			);
+		};
+
+		values.objectValidationRuleSettings.forEach(
+			(objectValidationRuleSetting) => {
+				const filteredObjectFieldByObjectValidationRuleSetting = filteredCustomObjectFields.find(
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/UniqueComposedKey.tsx:180]: Rename to filteredObjectFieldObjectValidationRuleSetting

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
Blocked by https://github.com/liferay-objects/liferay-portal/pull/2596.

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
resent [here](https://github.com/liferay-objects/liferay-portal/pull/2688)

**Pull request closed without merging**